packages/sdks/web-component/src/lib/helpers/helpers.ts

```diff
@@ -57,6 +57,11 @@ function resetUrlParam(paramName: string) {
   }
 }
 
+const getFlowIdFromExecId = (executionId: string) => {
+  const regex = /(.*)\|#\|.*/;
+  return regex.exec(executionId)?.[1];
```

Comment:
can it fail if length is 0 or 1?

Reply:
No, the regex.exec return an array or null
Changed to return an empty string in this case